tests/Fixtures/Integration/priority/no_duplicate_semicolons,switch_case_semicolon_to_colon.test

```diff
@@ -1,17 +0,0 @@
---TEST--
-Integration of fixers: no_duplicate_semicolons,switch_case_semicolon_to_colon.
```

Comment:
shoudln't this be converted to testing the integration of ``switch_case_semicolon_to_colon`` with the new fixer ?